examples/colab/image_enhancing.ipynb

```diff
@@ -0,0 +1,444 @@
+{
```

Comment:
<p>It is still above the license, please move below.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/hub/pull/352/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Oops, okay.</p><p> </p>